File: cpp/n242_isAnagram.cpp
// https://leetcode-cn.com/problems/valid-anagram/
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

class Solution {
public:
    bool isAnagram(string s, string t) {
        sort(s.begin(), s.end());
        sort(t.begin(), t.end());
        return s == t;
    }
};

int main(){
    return 0;
}

File: python/n242_isAnagram.py
# https://leetcode-cn.com/problems/valid-anagram/
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        l = list(s)
        l.sort()
        s = "".join(l)
        l = list(t)
        l.sort()
        t = "".join(l)
        return s == t
        
        
if __name__ == "__main__":
    solu = Solution()
    s = "anagram"
    t = "nargama"
    print(solu.isAnagram(s, t))
        